components/gallery/List.vue

```diff
@@ -0,0 +1,303 @@
+<template>
+    <div class="gallery-list">
+        <ul class="list list-main">
```

Comment:
I suspect you don't need both of this elements. Probably can be done with just the `ul`.

Reply:
I needed `.galler-list` to vertically center the hover image and list of items. Also to set the outer width to 1280px. The  `.list-main` class I needed  to handle the list columns layout and set max-width to 1180px. Totally cool if there's a better way to handle this but it's pretty much what I landed on.